style.css

```diff
@@ -411,6 +492,98 @@ ul {
   color: #fff;
 }
 
+@media screen and (min-width: 768px) {
+  .card1 {
+    width: auto;
+    display: grid;
+    grid-template-areas:
+      "work-title grid-line grid-line grid-line grid-line"
+      "photo1 photo1 photo1 p1-title p1-title"
+      "photo1 photo1 photo1 p1-text p1-text"
+      "photo1 photo1 photo1 list-1 list-1"
+      "photo1 photo1 photo1 button1 button1"
+      "photo1 photo1 photo1 . .";
+    height: 100%;
+    margin-top: 72px;
+    padding-top: 50px;
+    padding-left: 30px;
+    padding-right: 30px;
+  }
+
+  .work-title {
+    grid-area: work-title;
+    align-self: end;
+    width: auto;
+  }
+
+  .grid-line {
+    grid-area: grid-line;
+    align-self: center;
+    width: 90%;
+    padding-left: 20px;
+  }
+
+  .photo1 {
+    grid-area: photo1;
+    width: auto;
+    padding-bottom: 0;
+    margin-left: 40px;
+  }
+
+  .p1-title {
+    grid-area: p1-title;
+  }
+
+  .p1-text {
+    grid-area: p1-text;
+  }
+
+  .list-1 {
+    grid-area: list-1;
+  }
+
+  .button1 {
+    grid-area: button1;
+    padding: 0;
+  }
+
+  #work-title {
+    padding-top: 0;
+    text-align: right;
+  }
+
+  #p1-title {
+    padding: 0;
+    font-size: 55px;
+    width: 500px;
+    height: 55px;
+  }
+
+  #p1-text {
+    padding: 0;
+    width: 500px;
+    height: 150px;
+    margin: 0;
+    line-height: 28px;
+    font-size: 25px;
+  }
+
+  .line-card1 {
+    width: 100%;
+  }
+
+  .img1 {
+    padding-top: 0;
+    padding-right: 0;
+    width: 70%;
```

Comment:
- [ ] [REQUIRED] Kindly increase the width of your image to take up to 60% of the page's width. The *Featured project* image should take 60% of the page's width as seen below, and the description takes the remaining 40%

| Figma design | Your design |
| --- | --- |
| ![Screenshot from 2022-03-24 22-35-52](https://user-images.githubusercontent.com/52934128/160014423-fccf3401-d0ad-4377-801d-93cce8b40c55.png) | ![Screenshot from 2022-03-24 22-36-52](https://user-images.githubusercontent.com/52934128/160014523-807a96b9-f63d-4f90-b3b0-8f3a510517e8.png) |